packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,12 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// {@macro flutter.material.appbar.automaticallyImplyLeading}
+  bool? automaticallyImplyLeading;
+
+  /// {@macro flutter.material.appbar.leadingWidth}
```

Comment:
I think the template of this macro needs to be updated to not reference local fields inside of app_bar.dart. Like leading -> AppBar.leading to fix the docs issue.

Reply:
i've updated this line to [AppBar.automaticallyImplyLeading] instead of macro.